.gitignore

```diff
@@ -29,3 +29,6 @@ fika-*.tar
 # parser combinators for new syntax. This avoids compilation of the
 # entire parser which gives me a better feedback loop.
 /test/support/test_parser*
+
+.dist/
+.history/
```

Comment:
Do we need this?

Reply:
Actual when I open my visual studio, these two folders are automatically created so ..

- .history is just keeping track of what current change we are doing 
- .dist is temporary folder

Reply:
Ah i see. Those are IDE and text editor user specific folder(s).
I don't think we need to add them to the repo.

Reply:
Yes, I'm gone remove them from gitignore